lua/nvim-treesitter/indent.lua

```diff
@@ -243,14 +242,17 @@ function M.get_indent(lnum)
           end
         else
           -- aligned indent
-          if (c_delim_node and c_is_last_in_line and
-              c_srow and o_srow ~= c_srow and c_srow < lnum - 1) then
+          if c_delim_node and c_is_last_in_line and c_srow and o_srow ~= c_srow and c_srow < lnum - 1 then
```

Comment:
pretty sure c_delim_node here isn't necessary cause if c_srow can only exist if c_delim_node is not nil, so checking for `c_srow` is enough

Reply:
done.